docs/modelserving/v1beta1/llm/huggingface/README.md

```diff
@@ -0,0 +1,69 @@
+# Deploy the Llama2 model with Hugging Face LLM Serving Runtime
+The Hugging Face LLM serving runtime implements a runtime that can serve Hugging Face LLM model out of the box. 
+
+In this example, we deploy a Llama2 model from Hugging Face by running an `InferenceService` with [Hugging Face Serving runtime](https://github.com/kserve/kserve/tree/master/python/huggingfaceserver). Based on the performance requirement for large language models, KServe chooses to perform the inference using a more optimized inference engine like [vLLM](https://github.com/vllm-project/vllm) for text generation models.
+
+### Serve the Hugging Face LLM model using vLLM
+KServe Hugging Face runtime by default uses vLLM to serve the LLM models for faster inference, higher throughput than Hugging Face API, implemented with paged attention, continous batching, optmized CUDA kernel. 
+You can still use `--disable_vllm` flag to fall back to perform the inference using Hugging Face API.
+
+=== "Yaml"
+
+    ```yaml
+    kubectl apply -f - <<EOF
+    apiVersion: serving.kserve.io/v1beta1
+    kind: InferenceService
+    metadata:
+      name: huggingface-llama2
+    spec:
+      predictor:
+        model:
+          modelFormat:
+            name: huggingface
+          args:
+          - --model_name=llama2
+          - --model_id=meta-llama/Llama-2-7b-chat-hf
```

Comment:
For Llama2 model, I assume it will require `HUGGING_FACE_HUB_TOKEN` to download the model from huggingface as it requires a license. I am not sure now, just want to confirm.